src/shared/components/post/post-form.tsx

```diff
@@ -582,8 +582,11 @@ export class PostForm extends Component<PostFormProps, PostFormState> {
       case "loading":
         return <Spinner />;
       case "success": {
-        const suggestedTitle = this.state.metadataRes.data.metadata.title;
-
+        let suggestedTitle = this.state.metadataRes.data.metadata.title;
+        // Clean up the title of any extra whitespace and replace &nbsp; with a space
+        if (suggestedTitle) {
+          suggestedTitle = suggestedTitle.trim().replace(/\s+/g, " ");
+        }
```

Comment:
This can be simplified somewhat:
```suggestion
        // Clean up the title of any extra whitespace and replace &nbsp; with a space
        const suggestedTitle = this.state.metadataRes.data.metadata.title?.trim().replace(/\s+/g, " ");
```

Up to you if you still want to keep the comment.